etc/apache2/vhosts.d/openqa-common.inc

```diff
@@ -79,3 +79,4 @@ ServerSignature On
 # TODO: try mojo 7.39 - https://github.com/kraih/mojo/commit/f9ff45e48f606
 SetEnv proxy-nokeepalive 1
 
+ProxyTimeout 300
```

Comment:
Please add a comment on top.

I don't know if this will work in general, any external proxy might still have timeouts longer. Probably we also need to ensure that our requests never take that long internally but that does not need to stop this PR here.